src/cleanup.c

```diff
@@ -0,0 +1,26 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   cleanup.c                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/12 15:59:11 by gwolf             #+#    #+#             */
+/*   Updated: 2023/11/20 11:26:33 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cleanup.h"
+
+void	ft_free_char_arr(char **array)
+{
+	size_t	i;
+
+	i = 0;
```

Comment:
I generally use size_t for array iterators, since an array position can never be negative. This version also supports arrays double the size since unsigned

Reply:
Yes